src/pages/workspaces/List.js

```diff
@@ -128,12 +143,24 @@ export const WorkspaceList = () => {
 
   const [sort, setSort] = useState({ field: 'lastModified', direction: 'desc' })
 
+  const [gcpPersistentDisks, setGcpPersistentDisks] = useState()
+
   useOnMount(() => {
     const loadFeatured = async () => {
       setFeaturedList(await Ajax().FirecloudBucket.getFeaturedWorkspaces())
     }
 
+    const loadPersistentDisks = async () => {
+      setGcpPersistentDisks(_.filter(({ cloudContext }) => isGcpContext(cloudContext),
+        await Ajax().Disks.list({
```

Comment:
You need error handling around this in case the Ajax call fails.

There should be a ticket to rip this code out once the migration period is over and a TODO in the code. We don't want to keep the loading of the disks in this file permanently.

Reply:
shouldn't the other request have error handling too?

Reply:
Yes